gridt/src/app/core/settings.service.spec.ts

```diff
@@ -180,14 +170,34 @@ describe("IdentityService_AuthSuccesfull", () => {
     )));
 
     // Obtain the settings
-    service.getUserSettings();
+    service.updateUserSettings();
 
     // See if disabler it set to true
     expect(service.isDisabled$).toBeObservable(
-      cold('(a|)', {a: false})
-    )
+      cold('(a)', {a: true})
+    );
   });
 
+  it('should display local settings when server settings are not available', () => {
+    // Simulate the server not being reachable.
+    httpClientStub.post.and.returnValue(throwError(
+      new HttpErrorResponse(
+        {
+          status: 400, statusText: "Bad Request",
+          error: { message: "Could not retreive settings from server." }
+        }
+    )));
+
+    // Populate localStorage
+    secStoreStub.get$.and.returnValue(of(mock_settings[0]));
+
+    // Obtain the settings
+    service.updateUserSettings();
+
+    // Make sure the local storage settings are returned
+    expect(service.the_user_settings$).toBeObservable(cold('a',{a: mock_settings[0]}))
+  })
+
 });
 
 describe("IdentityService_AuthFailed", () => {
```

Comment:
Sometimes my test suite throws an error when loading this test suite and some times it does not:
![pic-selected-200907-1007-50](https://user-images.githubusercontent.com/1318056/92363606-067f9d80-f0f2-11ea-8b37-87b2bc071791.png)